geomstats/geometry/discrete_curves.py

```diff
@@ -1842,6 +1843,244 @@ def horizontal_path(t):
 
         return horizontal_path
 
+    def _dynamic_programming(
+            self, initial_curve, end_curve, n_discretization=100, max_slope=6):
+        r"""Compute the dynamic programming algorithm.
+
+        Find the reparametrization gamma of end_curve that minimizes the distance
+        between initial_curve and end_curve reparametrized by gamma, and output
+        the corresponding distance, using a dynamic programming algorithm.
+
+        The objective can be expressed in terms of square root velocity (SRV)
+        representations: it is equivalent to finding the gamma that maximizes
+        the L2 scalar product between initial_srv and end_srv@gamma where initial_srv
+        is the SRV representation of the initial curve and end_srv@gamma is the SRV
+        representation of the end curve reparametrized by gamma, i.e
+        .. math::
+        end_srv@\gamma(t) = end_srv(\gamma(t))\cdot\lvert\gamma(t)\rvert^\frac{1}{2}
```

Comment:
Maybe replace "\lvert\gamma(t)\rvert" by "| \gamma(t) |" which gives the same output in Latex but is more readable in the docstring ?